Error ------------------------------------------------------------------------------------------------------ test.js:4:5

Cannot declare `f` [1] because the name is already bound. [name-already-bound]

   test.js:4:5
   4| var f: number = 3; // This assignment should not trigger a check for the
          ^

References:
   test.js:2:10
   2| function f(): void {}
               ^ [1]



Found 1 error
